import-scripts/update-sophia-mskimpact.sh

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env bash
+
+# File containing list of patients should be passed in as argument
+export SUBSET_FILE="$1"
+export CANCER_TYPE="$2"
+export CURRENT_DATE="$(date '+%m.%d.%y')"
+export COHORT_NAME="sophia-$CANCER_TYPE-data-$CURRENT_DATE"
+
+export SOPHIA_REPO_NAME="sophia-data"
+export SOPHIA_DATA_HOME="$PORTAL_DATA_HOME/$SOPHIA_REPO_NAME"
+export SOPHIA_MSKIMPACT_STABLE_ID="sophia_mskimpact"
+export SOPHIA_COHORT_HOME="$SOPHIA_DATA_HOME/deliveries/$COHORT_NAME"
+export SOPHIA_MSK_IMPACT_DATA_HOME="$SOPHIA_COHORT_HOME/$SOPHIA_MSKIMPACT_STABLE_ID"
+export SOPHIA_TMPDIR="$SOPHIA_COHORT_HOME/tmp"
```

Comment:
so if I understand right -- 

sophia_portal_data/deliveries/<cancer-type>-<datespecific>/sophia_mskimpact

Is there anything parallel to deliveries? 

Also for tmp function, let's use something on the tmp volume - think we autogenerate a tmp directory as part of the script. If that's here, wil we still need the `sophia_mskimpact` directory?

Reply:
Was thinking something closer to 

cbio-portal-data/delivery-pipelines/<recepient e.g. sophia>/<cancer-type-date>

Reply:
Right now `/data/portal-cron/cbio-portal-data/sophia-data` contains these directories/files:
`cohort_ids/`: contains files with list of IDs for each cohort
`deliveries/`: contains directories for each cancer type cohort and a date (in the case that we have to make changes / deliver multiple versions)
- for example, a subdirectory would be `sophia-lung-data-01.28.24` and that directory contains: `README.pdf`, `gene_panels/` directory, and `sophia-mskimpact/` directory containing study files. This structure was copied from the structure of the az repo but since sophia doesn't import these files to a cbio instance it's probably not totally necessary, just thought it was a good idea to keep the same structure
`gene_panels/`: gene panel files that are packaged into each delivery
`logs/`: log files from cohort generation
`README.pdf`: readme file describing files that is packaged into each delivery

if you want to talk through this setup @averyniceday let me know, totally open to opinions